### ISSUE TITLE
Compiled by a pre-release version of Kotlin

### ISSUE THREAD

**Issue by mradzinski** [MarkusAmshove/Kluent#23]
```Class 'org.amshove.kluent.AssertionsKt' is compiled by a pre-release version of Kotlin and cannot be loaded by this version of the compiler```

Using Kotlin 1.1.0-rc-91. You might probably want to have separate branches/releases, one for Kotlin 1.0.6 and another for 1.1.0 which is about to hit stable version.

**Comment by MarkusAmshove** [comment]
Hi,

I've read through the release notes (https://blog.jetbrains.com/kotlin/2017/02/kotlin-1-1-release-candidate-is-here/ and https://blog.jetbrains.com/kotlin/2017/01/kotlin-1-1-beta-is-here/) and I can't find anything that is going to deprecated that we use.

Can you try the new released version 1.15?

I'll have a look myself the next days

**Comment by mradzinski** [comment]
It's not due to deprecations but a mismatch on the compiler versions. I think the generated classes have something to do with the error. If you check my fork you'll see that I only updated the kotlin version in the gradle build file and everything got fixed, so I believe changing that in your code should do the trick.

**Comment by MarkusAmshove** [comment]
As Kotlin 1.1 was released today I created a new project using Kotlin 1.1 and referenced the current Kluent version (1.15) which was compiled with 1.06.

I don't get an error, so I guess its okay now?

**Comment by kazufukurou** [comment]
```
:prepareDebugUnitTestDependenciesConflict with dependency 'org.jetbrains.kotlin:kotlin-stdlib'. Resolved versions for app (1.1.0) and test app (1.0.6) differ. See http://g.co/androidstudio/app-test-app-conflict for details.
FAILED
```

**Comment by kazufukurou** [comment]
Adding `testCompile "org.jetbrains.kotlin:kotlin-stdlib:1.1.0"` to project dependencies fixed the issue.

**Comment by MarkusAmshove** [comment]
I'll have the kotlin version updated in the next release, which will likely be done today to fix #27

**Comment by MarkusAmshove** [comment]
I upgraded the Kotlin version to 1.1 with v1.16
